apps/preview/next/pages/showcases/FormFields/FormFieldsRequired.tsx

```diff
@@ -0,0 +1,697 @@
+  },
+  {
+    label: 'Home Decor',
+    value: 'Home Decor',
+  },
+  {
+    label: 'Technology and Gadgets',
+    value: 'Technology and Gadgets',
+  },
+  {
+    label: 'Beauty and Cosmetics',
+    value: 'Beauty and Cosmetics',
+  },
+  {
+    label: 'Outdoor and Adventure',
+    value: 'Outdoor and Adventure',
+  },
+  {
+    label: 'Parenting and Baby Products',
+    value: 'Parenting and Baby Products',
+  },
+  {
+    label: 'Books and Literature',
+    value: 'Books and Literature',
+  },
+  {
+    label: 'Sports and Fitness',
+    value: 'Sports and Fitness',
+  },
+  {
+    label: 'Food and Cooking',
+    value: 'Food and Cooking',
+  },
+];
+const options = [
+  {
+    label: 'Afghanistan',
+    value: 'afghanistan',
+  },
+  {
+    label: 'Albania',
+    value: 'albania',
+  },
+  {
+    label: 'Angola',
+    value: 'angola',
+  },
+  {
+    label: 'Bahamas',
+    value: 'bahamas',
+  },
+  {
+    label: 'Bangladesh',
+    value: 'bangladesh',
+  },
+  {
+    label: 'Canada',
+    value: 'canada',
+  },
+  {
+    label: 'Chile',
+    value: 'chile',
+  },
+  {
+    label: 'Czech Republic',
+    value: 'czech Republic',
+  },
+  {
+    label: 'Colombia',
+    value: 'colombia',
+  },
+  {
+    label: 'Congo',
+    value: 'congo',
+  },
+  {
+    label: 'Croatia',
+    value: 'croatia',
+  },
+  {
+    label: 'Cuba',
+    value: 'cuba',
+  },
+  {
+    label: 'Denmark',
+    value: 'denmark',
+  },
+  {
+    label: 'Dominica',
+    value: 'dominica',
+  },
+  {
+    label: 'Egypt',
+    value: 'egypt',
+  },
+  {
+    label: 'Ethiopia',
+    value: 'ethiopia',
+  },
+  {
+    label: 'Estonia',
+    value: 'estonia',
+  },
+];
+const areaCodes = [
+  { label: '1', value: '1' },
+  { label: '20', value: '20' },
+  { label: '45', value: '45' },
+  { label: '53', value: '53' },
+  { label: '56', value: '56' },
+  { label: '57', value: '57' },
+  { label: '93', value: '93' },
+  { label: '243', value: '243' },
+  { label: '244', value: '244' },
+  { label: '251', value: '251' },
+  { label: '355', value: '355' },
+  { label: '372', value: '372' },
+  { label: '385', value: '385' },
+  { label: '420', value: '420' },
+  { label: '880', value: '880' },
+];
+
+export default function FormFields() {
+  const [personalInformation, setPersonalInformation] = useState(true);
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [website, setWebsite] = useState('');
+  const [areaCode, setAreaCode] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [description, setDescription] = useState('');
+  const [emailNotifications, setEmailNotifications] = useState(['safety-alerts']);
+  const [pushNotifications, setPushNotifications] = useState('');
+  const [searchValue, setSearchValue] = useState<string>('');
+  const [selectedValueCombobox, setSelectedValueCombobox] = useState<string>('');
+  const [isValid, setIsValid] = useState<boolean | undefined>(undefined);
```

Comment:
question: isn't default value be `undefined` so no need to define it ?
```suggestion
  const [isValid, setIsValid] = useState<boolean | undefined>();
```

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2787/commits/4340777142524b2132252c34501b858df32e129f